### PR TITLE
Announce iviewer 0.3.0

### PR DESCRIPTION
**Pull request by hflynn**
Content based on https://github.com/ome/omero-iviewer/pull/124/

cc @jburel

### CODE REVIEW

**Comment by hflynn** [comment]
staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/2017/09/26/iviewer-0-3-0.html

**Comment by hflynn** [comment]
Ah, is the plan to have a new demo movie @jburel ? If so I can update the link once we have a URL or alternatively I can just link to the channel at https://www.youtube.com/channel/UCyySB9ZzNi8aBGYqcxSrauQ and let users find it and the 0.1.0 movie themselves

**Comment by jburel** [comment]
Not for this version
I think we will do new movie when we release the next one since we will be very closed to insight/measurement tool replacement

**Comment by hflynn** [comment]
Great, should be fine as is then

File: _posts/2017-09-26-iviewer-0-3-0.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Today we are releasing our version 0.3.0 of OMERO.iviewer: an OMERO.web app
+for visualizing images in OMERO. This version requires OMERO 5.3.3 or newer.
+
+This 0.3.0 version is still work-in-progress and some areas are likely to be
+modified.
+
+This version includes:
+
+- support for saving the image in the viewport as a .png file
+- support for exporting basic ROIs measurement (area and length) in a CSV file
```
Comment (mtbc) [inline _posts/2017-09-26-iviewer-0-3-0.md]: maybe something like "... exporting basic measurements (area and length) of ROIs in ..."?

File: _posts/2017-09-26-iviewer-0-3-0.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Release of OMERO.iviewer 0.3.0
+intro-blurb: The OME team is pleased to announce the release of OMERO.iviewer 0.3.0
+---
+Today we are releasing our version 0.3.0 of OMERO.iviewer: an OMERO.web app
```
Comment (will-moore) [inline _posts/2017-09-26-iviewer-0-3-0.md]: "our" version? Are there any others? Remove "our".

File: _posts/2017-09-26-iviewer-0-3-0.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+for visualizing images in OMERO. This version requires OMERO 5.3.3 or newer.
+
+This 0.3.0 version is still work-in-progress and some areas are likely to be
```
Comment (will-moore) [inline _posts/2017-09-26-iviewer-0-3-0.md]: "a work-in-progress"?

File: _posts/2017-09-26-iviewer-0-3-0.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+modified.
+
+This version includes:
+
+- support for saving the image in the viewport as a .png file
+- support for exporting basic ROIs measurement (area and length) in a CSV file
+- support for opening datasets and wells
```
Comment (will-moore) [inline _posts/2017-09-26-iviewer-0-3-0.md]: "Datasets and Wells" (camel case for actual OMERO objects)

**Comment by will-moore** [comment]
Looking good. Just a few minor comments/suggestions.

**Comment by hflynn** [comment]
Fixes pushed, thanks both

**Comment by hflynn** [comment]
Merging for @jburel to release live

**Pull request merged**